qiskit_nature/second_q/mappers/__init__.py

```diff
@@ -93,6 +93,7 @@
 from .spin_mapper import SpinMapper
 from .vibrational_mapper import VibrationalMapper
 from .interleaved_qubit_mapper import InterleavedQubitMapper
+from .tapered_qubit_mapper import TaperedQubitMapper
```

Comment:
Just a quick comment: having fixed the merge conflict here just now I noticed that the docstring in this file is not being updated to include the `TaperedQubitMapper`.
Please add a new section to include that.

PS: I will do another full review later today :+1: